app/api/services/briefs.py

```diff
@@ -79,3 +79,49 @@ def get_assessors(self, brief_id):
                    .all())
 
         return [r._asdict() for r in results]
+
+    def get_briefs_by_filters(self, status=[], open_to=[], brief_type=[]):
```

Comment:
Be careful with mutable defaults: https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument